src/vivarium_public_health/risks/implementations/low_birth_weight_and_short_gestation.py

```diff
@@ -371,11 +371,9 @@ def register_target_modifier(self, builder: Builder) -> None:
 
     def get_age_intervals(self, builder: Builder) -> dict[str, pd.Interval]:
         age_bins = builder.data.load("population.age_bins").set_index("age_start")
-        exposure = builder.data.load(f"{self.risk}.exposure")
-        exposure = exposure[exposure["age_end"] > 0]
-
+        relative_risks = builder.data.load(f"{self.risk}.relative_risk")
```

Comment:
Do we already have this data in this component? We may not need to reload it from the artifact.

Reply:
Technically we load it in the RiskEffect component for the RR lookup table so not in a form we can use for this.